src/screens/Learn/Updates/UpdatesLanding.tsx

```diff
@@ -3,18 +3,20 @@ import { formatMetatagDate } from 'common/utils';
 import ArticlesLanding from '../Articles/ArticlesLanding';
 import articles from 'cms-content/updates';
 
+const metaDescription = `Explore recent developments — including new tools and simple instructions on how to use them — at Covid Act Now.`;
```

Comment:
FWIW-
1. This description feels pretty unwieldy.
2. I think the correct punctuation here is an `&mdash;` instead of a normal dash. And it's [unclear](https://prowritingaid.com/grammar/1000066/Should-I-use-spaces-around-an-em-dash) if there should be spaces around it or not.  We'd have to ask Karen probably. 😁 

Neither is a big deal and I assume you didn't write this copy, so feel free to leave as-is.